README.md

```diff
@@ -155,16 +155,22 @@ more insights.
 ## Supported languages
 
 Official Kaitai Struct [compiler] now supports compiling `.ksy` into
-source modules for the following languages:
+source modules for [the following languages](https://github.com/kaitai-io?q=runtime&type=&language=):
 
+* C++
 * C#
+* Go
 * Java
 * JavaScript
+* TypeScript
 * Lua
+* Perl
 * Nim
 * PHP
 * Python
 * Ruby
+* Swift
+* Rust
```

Comment:
> Official Kaitai Struct [compiler] now **supports compiling `.ksy` into source modules** for the following languages:

TypeScript, Swift and Rust shouldn't be there, because the KS compiler doesn't support them yet (see `kaitai-struct-compiler --help`):

```
  -t, --target <language>  target languages (graphviz, csharp, rust, all, perl, java, go, cpp_stl, php, lua, python, nim, html, ruby, construct, javascript)
```

Support of these targets in Kaitai Struct has not gone too far - work started just on their runtime libraries and even these are often incomplete or dysfunctional.

I suggest using the same list as on the [kaitai.io homepage (section _Language-neutral_)](https://kaitai.io/#:~:text=Language-neutral), preferably keeping the _wrench icons_ to show that Go and Nim are in the state of work-in-progress. This list should be up to date.

[compiler]: https://github.com/kaitai-io/kaitai_struct_compiler